app/models/departure_fetcher.rb

```diff
@@ -1,16 +1,21 @@
+require 'duration'
+
 class DepartureFetcher
   include ActiveModel::SerializerSupport
 
   def initialize(params)
     @time = params[:time] ? Time.zone.parse(params[:time]) : Time.current
     @stop_id = params[:stop_id]
+    @time_limit = params.fetch(:time_limit, 10)
+
+    @active_duration = Duration.new((-1 * @time_limit.minutes))
```

Comment:
I thought about that, but kind of didn't like the idea of having it need to be negative. Mainly because the idea of a positive limit doesn't make sense. e.g. If some specified 10 we would not show things leaving in less than 10 minutes.